### PR TITLE
Add .env for deployer setup

### PR DESCRIPTION
**Pull request by ishantanu**
Adding `.env` file for setting up the WordPress specific environment variables.

### CODE REVIEW

File: multiple-sites/.env.deployer [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Generate your keys here: https://roots.io/salts.html
+AUTH_KEY='generateme'
+SECURE_AUTH_KEY='generateme'
+LOGGED_IN_KEY='generateme'
+NONCE_KEY='generateme'
+AUTH_SALT='generateme'
+SECURE_AUTH_SALT='generateme'
+LOGGED_IN_SALT='generateme'
+NONCE_SALT='generateme'
+WP_CACHE_KEY_SALT='example.com:' #trailing `:` is important
```
Comment (rahul286) [inline multiple-sites/.env.deployer:22]: `WP_CACHE_KEY_SALT` should have been on a separate line with a comment explaining the purpose.

The way it looks now its part of https://roots.io/salts.html

Comment (rahul286) [reply multiple-sites/.env.deployer:22]: @iamShantanu101 please look into it

**Comment by rahul286** [comment]
@mbtamuli why we want to mix deployer in this project? I believe the focus of this should have been `hosting` alone for a while.

File: multiple-sites/.env.deployer (hunk 1 above)
Comment (ishantanu) [reply multiple-sites/.env.deployer:22]: Updated with description #18

**Pull request merged**